buildSrc/src/main/kotlin/non-multiplatform.publish-conventions.gradle.kts

```diff
@@ -5,27 +5,16 @@ plugins {
 }
 
 // MPP projects pack their sources automatically, java libraries need to explicitly pack them
-val sourcesJar = tasks.register<Jar>("sourcesJar") {
+val sourcesJar by tasks.registering(Jar::class) {
     archiveClassifier.set("sources")
     from("src/main/kotlin")
 }
-
 publishing {
     // Configure java publications for non-MPP projects
     publications {
-        if (project.name == "atomicfu-gradle-plugin") {
```

Comment:
I've just removed this logic, because `from(components["java"])` does not fail, even without `java` component, as I see